src/Core/Impl/IO/DirectoryInfoProxy.cs

```diff
@@ -40,6 +43,27 @@ public sealed class DirectoryInfoProxy : IDirectoryInfo {
                 .EnumerateFileSystemInfos()
                 .Select(CreateFileSystemInfoProxy);
 
+        public IEnumerable<IFileSystemInfo> EnumerateFileSystemInfos(string[] includeFiles, string[] excludeFiles) {
+            Matcher matcher = new Matcher();
```

Comment:
https://github.com/Microsoft/python-language-server/issues/635, just for the reference.